Clarify Worker activity limit options (#1138)
temporalio/documentation: docs/operation/how-to-tune-workers.md

### Rate Limiting

> If, after adjusting the poller and executors count as specified above, you still observe an elevated `schedule_to_start`, underutilized Worker hosts, or high `worker_task_slots_available`, you may want to check:

Reviewer: ```suggestion
If, after adjusting the poller and executors count as specified earlier, you still observe an elevated `schedule_to_start`, underutilized Worker hosts, or high `worker_task_slots_available`, you might want to check the following:
```

> 1. If server-side rate limiting per Task Queue is set by `WorkerOptions#maxTaskQueueActivitiesPerSecond`, remove the limit or adjust the value up. (See [Go](/docs/go/how-to-set-workeroptions-in-go/#taskqueueactivitiespersecond) and [Java](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/worker/WorkerOptions.Builder.html).)

Reviewer: Should this list be unordered instead of ordered? It doesn't appear to be a sequence.

Author: I think unordered. Edited.
